docs/getting_started/dashboard.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Console Dashboard
+sidebar_position: 3
+---
+
+The [Console](https://app.calimero.network/dashboard) dashboard is your gateway to managing your Calimero Private Shard. Here you can access all the information about your nodes, indexers and services. The dashboard also provides users with access to important information such as the RPC and Indexer GraphQL URL endpoints, as well as basic statistics about the network, including current block height, number of transactions, and number of accounts.
+
+If your infrastructure needs grow, you can use the Console to scale up the number of validators, RPCs, and indexers running on your network. This ensures that your network can handle increased traffic as your business grows.
+
+- **Environment**: This refers to either Testnet or Mainnet blockchain environment.
```

Comment:
```suggestion
- **Environment**: This refers to either Testnet or Mainnet environment.
```
Blockchain is unnecessary.
It's difficult to convey this info, but we should try to somehow note that testnet/mainnet affects only the bridging part.
If the shard is used without going to the public blockchain, then the difference is just in the shard name (does it have `-testnet` suffix or not)

Reply:
let me do this in a seperate PR